lms/templates/edxnotes/edxnotes.html

```diff
@@ -0,0 +1,93 @@
+<%inherit file="/main.html" />
+
+<%block name="pagetitle">${_("Student Notes")}</%block>
+<%block name="headextra">
+    <%static:css group='style-course'/>
+</%block>
+
+<%include file="/courseware/course_navigation.html" args="active_page='edxnotes'" />
+<section class="container wrapper-student-notes">
+    <div class="info-wrapper">
+        <section class="updates">
+            <div class="title-search-container">
+                <div class="wrapper-title">
+                    <h1 class="page-title">${_('Notes')} <small class="page-subtitle">${_("Highlights and notes you've made in course content")}</small></h1>
+                </div>
+            % if notes:
+                <div class="wrapper-notes-search">
+                    <form role="search" action="${search_endpoint}" method="GET" id="search-notes-form" class="is-hidden">
+                        <label for="search-notes-input" class="sr">${_('Search notes for:')}</label>
+                        <input type="search" class="search-notes-input" id="search-notes-input" name="note" placeholder="${_('Search notes for...')}">
+                        <button type="submit" class="search-notes-submit"><i class="icon-search"></i> ${_('Search')}</button>
+                    </form>
+                </div>
+            % endif
+            </div>
+
+            <section class="wrapper-tabs">
+                <div class="tab-list is-hidden">
+                    <h2 id="tab-view" class="tabs-label">${_('View notes by:')}</h2>
+                </div>
+                <div class="inline-error is-hidden" tabindex="-1"></div>
+            % if notes:
+                <div class="ui-loading" tabindex="-1">
+                    <p><span class="spin"><i class="icon-refresh"></i></span> <span class="copy">${_("Loading...")}</span></p>
+                </div>
+            % else:
+                <section class="is-empty">
+                    <article>
+                        <h2>${_('You have not made any notes in this course yet.')}</h2>
+                        <p>${_('Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.')}</p>
+
+                        <h2>${_('Other Students In This Course Are Using Notes To:')}</h2>
+                        <ul>
+                            <li>${_('Add study notes to course content for exams and homework.')}</li>
+                            <li>${_('Highlight important concepts for later coursework or future courses')}</li>
+                            <li>${_('Donec id elit non mi porta gravida et eget metus.')}</li>
```

Comment:
@andy-armstrong
>More placeholder text here...

Already removed the placeholder text in the `li` tag (see comments to @srpearce below)

@srpearce Should we add some text to replace the current placeholder text in the `p` tag? Or should it be blank and I just remove the `p` tag altogether?